builds/e2e/longhaul.yaml

```diff
@@ -260,6 +260,7 @@ jobs:
           downloadPath: '$(Build.StagingDirectory)'
           artifactName: $(images.artifact.name.windows)
           itemPattern: |
+            $(images.artifact.name.windows)/CACertificates/*
```

Comment:
Latest update in runE2Etests.ps1 committed to master; we should not need CACertificates files for stress or long haul test.

Reply:
Please get the latest runE2ETests.ps1 and verify.  thanks.